weather_mv/loader_pipeline/util_test.py

```diff
@@ -41,5 +42,68 @@ def test_no_duplicate_coordinates(self):
 
 
 class ToJsonSerializableTypeTests(unittest.TestCase):
-    # TODO(#106): Write tests...
-    pass
+
+    def assert_value_and_type(self,input,output):
+        # check if it retains the original value
+        self.assertEqual(input,output)
+
+        # check if it retains the original type
+        self.assertEqual(type(input), type(output))
+
+    def assert_serializes_json(self, input):
+        try:
+            json.loads(json.dumps(input))
+            print("Successfully passed!")
+        except json.JSONDecodeError:
+            self.fail()
+
+    def test_numpy_float_becomes_float(self):
+        input = np.float(42)
+        
+        # genrate output from the function being tested
+        output = to_json_serializable_type(input)
+
+        # assert the output with set of given conditions
+
+        # check if it is serialized in json 
+        self.assert_serializes_json(output)
+
+        # check if it retains the original value and type
+        self.assert_value_and_type(input,output)
+       
+
+    def test_numpy_array(self):
+        input = np.array([1, 2, 3, 4, 5777, 7, 4252, 5426])
+        output = to_json_serializable_type(input)
+        self.assert_serializes_json(output)
+        self.assert_value_and_type(input,output)
+
+    def test_strings(self):
+        input = "hello there"
+        output = to_json_serializable_type(input)
+        self.assert_serializes_json(output)
+        self.assert_value_and_type(input,output)
+
+    def test_date_and_time(self):
+        input = datetime.datetime.now().isoformat()
```

Comment:
It would be a more powerful test if you also exercised serializing a `datetime.datetime`, `np.datetime64`, etc. in addition to an ISO string. As mentioned above, you will likely not be able to use the `assert_value_and_type` method.